CONTRIBUTING.md

```diff
@@ -110,7 +110,7 @@ DevOps Dojo development is managed with one repository:
 To prepare your dedicated GitHub repository:
 
 1. Fork in GitHub [https://github.com/dxc-technology/online-devops-dojo](https://github.com/dxc-technology/online-devops-dojo)
-2. Clone *your forked repository* (e.g. [https://github.com/jdoe1000/online-devops-dojo](https://github.com/jdoe1000/online-devops-dojo)
+2. Clone *your forked repository* (e.g., `https://github.com/jdoe1000/online-devops-dojo`)
```

Comment:
Made these code ticks because there's no need to link to a page that doesn't actually exist (it's just a reference example)

Reply:
The other URL in the page are in normal text, not in code.
I prefer
h\&#8203;ttps://github.com/<username\\>/online-devops-dojo
which renders like
h&#8203;ttps://github.com/<username\>/online-devops-dojo